docker/release/fedora24/Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM fedora:24
 # TODO: Until a release of PowerShell for Fedora 24 is available,
 # this Dockerfile installs the CentOS 7 version for compatibility.
 
-ARG POWERSHELL_VERSION=6.0.0-beta.7
-ARG POWERSHELL_RELEASE=v6.0.0-beta.7
-ARG POWERSHELL_PACKAGE=powershell-6.0.0_beta.7-1.el7.x86_64.rpm
+ARG POWERSHELL_VERSION=6.0.0-beta.8
+ARG POWERSHELL_RELEASE=v6.0.0-beta.8
+ARG POWERSHELL_PACKAGE=powershell-6.0.0_beta.8-1.rhel.7.x86_64.rpm
```

Comment:
According to Linux.md we support fedora 25 and 26. Should this file be called /fedora25/Dockerfile?

Reply:
fedora 24 has reached the end of lifecycle. I will remove this file and create new docker files for 25 and 26 in a separate PR.

Reply:
Sounds good.